File: src/main/java/com/bajdcc/LALR1/interpret/os/ui/UIMonitor.java
package com.bajdcc.LALR1.interpret.os.ui;

import com.bajdcc.LALR1.interpret.os.IOSCodePage;
import com.bajdcc.util.ResourceLoader;

/**
 * 【界面】监视器
 *
 * @author bajdcc
 */
public class UIMonitor implements IOSCodePage {
	@Override
	public String getName() {
		return "/ui/monitor";
	}

	@Override
	public String getCode() {
		return ResourceLoader.INSTANCE.load(getClass());
	}
}

File: src/main/java/com/bajdcc/LALR1/interpret/os/ui/UIMonitor.kt
package com.bajdcc.LALR1.interpret.os.ui

import com.bajdcc.LALR1.interpret.os.IOSCodePage
import com.bajdcc.util.ResourceLoader

/**
 * 【界面】监视器
 *
 * @author bajdcc
 */
class UIMonitor : IOSCodePage {
    override val name: String
        get() = "/ui/monitor"

    override val code: String
        get() = ResourceLoader.load(javaClass)
}
